change-beta/@azure-communication-react-fcbe2e27-0e50-49c3-93ae-f29e07a63ad2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "make sure call readiness works on landscape",
```

Comment:
nit: Center Call Readiness landscape mode

Reply:
Those changes center the call readiness modal also remove the scroll bars